dbOperations/hooks/useUserInfo.js

```diff
@@ -31,7 +32,14 @@ export const useUserInfo = ({
           doc(db, "teams", currentTeamId, "users", userId),
         );
         const data = querySnapshot.data();
-        if (!data) {
+
+        // DEBUG REMOVE BELOW LOG LINES LATER
```

Comment:
Ahh looks like you already have this planned lol

Reply:
should prob remove it in this pr

Reply:
Should be resolved by https://github.com/Golf-Drill-Challenge-App/Golf-App/pull/288/commits/69b6322cfc0d8564b927f921b64c41ba3a58fe00